addons/website_event_sale/i18n/website_event_sale.pot

```diff
@@ -193,3 +193,8 @@ msgstr ""
 #: model_terms:ir.ui.view,arch_db:website_event_sale.modal_ticket_registration
 msgid "to"
 msgstr ""
+
+#. module: website_event_sale
+#: model_terms:ir.ui.view,arch_db:website_event_sale.modal_ticket_registration
+msgid "Tax Excluded"
+mgstr ""
```

Comment:
```suggestion
msgstr ""
```

Runbot was unhappy. Also, pretty sure it should include the parentheses.

There's a translation export tool in settings, in dev mode, at the top. Maybe check against that :)

Reply:
lol thank you- not sure why I thought I could just improvise that :facepalm: